src/inverseRobot.js

```diff
@@ -25,6 +25,17 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const resultRobot = {};
```

Comment:
Незрозуміла назва змінної, що таке result robot ? Робот результату ?
```suggestion
  const inversedRobot = {};
```